doc/tutorials/dune-package-management/locking.md

```diff
@@ -0,0 +1,66 @@
+# Locking Your Dependencies
+
+In the default use-case Dune will automatically determine which packages, their
+versions and their recursive dependencies to install. However, given that new
```

Comment:
The first sentence was hard for me to parse. Could something like this make it clearer?

```suggestion
In the default use-case Dune will automatically determine which packages to
install, including solving compatible versions and installing their dependencies
recursively. However, given that new
```